### PR TITLE
feat: add calc-macos

### PR DESCRIPTION
**Pull request by Tashima42**

### CODE REVIEW

**Comment by federico-terzi** [comment]
Hey @Tashima42,

Thanks for your help :) The folder structure is not correct, please check out the report from the CI and how other packages are structured

Cheers :)

**Comment by Tashima42** [comment]
> Hey @Tashima42,
> 
> Thanks for your help :) The folder structure is not correct, please check out the report from the CI and how other packages are structured
> 
> Cheers :)

Hey, @federico-terzi. Done!

File: packages/calc-macos/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# espanso-calc-macos
+Basic calculation pacakge for epanso that works in MacOS.
```
Comment (katrinleinweber) [inline packages/calc-macos/0.1.0/README.md:2]:
```suggestion
Basic calculation pacakge for espanso that works in macOS.
```

**Comment by federico-terzi** [comment]
@Tashima42 @katrinleinweber Thanks for the help! Looks great 🚀

**Pull request merged**